### ISSUE TITLE
Major graphics bugs after 7.32e

### ISSUE THREAD

**Issue by Halifay** [ValveSoftware/Dota2-Gameplay#7445]
### Description

Description
I had decent fps prior to the last update. Just after it, flickering horizontal lines appeared. They make the interface temporarily transparent including the skills panel or main menu. It also goes with considerable fps reduction. Notably that I can't make a screenshot or record it on the video, these bugs become invisible. If I try to screenshot they just disappear on the second and then reappear.
The second problem that I can show is that Muerta's presenting scene has bugged particles and during her ghost form the video slows fps extremely.
My system is Ubuntu 20.04.5 LTS. My graphics settings are the lowest possible.

### Screenshots

![Screenshot from 2023-03-07 21-44-20](https://user-images.githubusercontent.com/29117268/224367873-77db093c-71b5-43f1-9007-01941e9f9fbe.png)
![Screenshot from 2023-03-07 21-44-25](https://user-images.githubusercontent.com/29117268/224367889-f9e58973-2945-42f7-a8bf-45b3273e7fe8.png)

**Comment by Gweenteaneko** [comment]
#7203

**Comment by Halifay** [comment]
> #7203

Doesn't seem to be the same problem as I have working fog of war and no shadows.

**Comment by shaileshgarg** [comment]
The Muerta intro works normal if Anti-alising and High Quality Dashboard, both are on.

**Comment by shaileshgarg** [comment]
https://www.reddit.com/r/DotA2/comments/11or5uk/comment/jbuvecs/?utm_source=share&utm_medium=web2x&context=3, try this for FPS

**Comment by Halifay** [comment]
> https://www.reddit.com/r/DotA2/comments/11or5uk/comment/jbuvecs/?utm_source=share&utm_medium=web2x&context=3, try this for FPS

Sadly this didn't resolve the problem with flickering menu. FPS seems to be on solid 60 and it wasn't a big issue before. Also I am not that much concerned with it because I can't play at all if ui just flickers all the time.

**Comment by Halifay** [comment]
> The Muerta intro works normal if Anti-alising and High Quality Dashboard, both are on.

Yep, now the scene seems ok, but the first phase of it still has like 5 fps, the second is smooth.
